anago

```diff
@@ -554,13 +557,20 @@ announce () {
   local announcement_text=/tmp/$PROG-announce.$$
 
   ((FLAGS_nomock)) || mailto=$USER@$DOMAIN_NAME
+  if [[ -n "${FLAGS_mailto}" ]]; then
+    mailto="${FLAGS_mailto}"
+  fi
 
   if [[ -n "$PARENT_BRANCH" ]]; then
     create_branch_announcement
   else
     create_announcement
   fi > $announcement_text
 
+  ((FLAGS_yes)) \
+   || common::askyorn -e "Pausing here. Confirm announce to $mailto" \
```

Comment:
By "yes", do you mean you prefer to only prompt if mailto was customized? I think it's helpful to always prompt at this stage because I like to have control over the timing of the announcement.